### DISCUSSION TITLE
Router in Controller

### DISCUSSION THREAD

**Discussion by devalexandre** [adonisjs/core#1943] [General]
In doc, show when use controller all routers is loaded in boot, bur when I use @get decorater , display error taht it don't is a function

```.js
export default class UsersController {
  @get('/users') // The route
  public async index() {
    return { name: 'Alexandre' }
  }
}
```

references:
[https://preview.adonisjs.com/guides/http/controllers#what-controllers-dont-do](https://preview.adonisjs.com/guides/http/controllers#what-controllers-dont-do)

**Comment by RomainLanz** [comment]
Hey @devalexandre! 👋 

The title of the section is "What controllers **DONT** do.

**Reply by devalexandre** [reply]
got it, I thought it was some new feature
